Improving C#/ASA type marshaling doc (#53542)
Azure/azure-content: articles/stream-analytics/stream-analytics-edge-csharp-udf-methods.md

The format of any UDF package has the path `/UserCustomCode/CLR/*`. Dynamic Link Libraries (DLLs) and resources are copied under the `/UserCustomCode/CLR/*` folder, which helps isolate user DLLs from system and Azure Stream Analytics DLLs. This package path is used for all functions regardless of the method used to employ them.

## Supported types and mapping

> For Azure Stream Analytics values to be used in C#, they need to be marshaled from one environment to the other. The Marshaling happens for all input parameters of an UDF. Every Azure Stream Analytics type has a coresponding type in C# shown on the table bellow:

Reviewer: ```suggestion
For Azure Stream Analytics values to be used in C#, they need to be marshaled from one environment to the other. Marshaling happens for all input parameters of a UDF. Every Azure Stream Analytics type has a corresponding type in C# shown on the table below:
```

|**Azure Stream Analytics type** |**C# type** |
|---------|---------|
|bigint | long |
|float | double |
|nvarchar(max) | string |
|datetime | DateTime |
|Record | Dictionary\<string, object> |
|Array | Array\<object> |

> The same is true when data needs to be marshaled from C# to Azure Stream Analytics, this happens on the output value of an UDF. The table bellow shows what types are supported:

Reviewer: ```suggestion
The same is true when data needs to be marshaled from C# to Azure Stream Analytics, which happens on the output value of a UDF. The table below shows what types are supported:
```
